src/casync-tool.c

```diff
@@ -673,7 +685,13 @@ static int parse_argv(int argc, char *argv[]) {
                 }
         }
 
-        /* Propagate our verbose setting to helpers we fork off */
+        /* Propagate some settings to helpers we fork off */
+        if (arg_log_level >= 0) {
+                char buffer[4];
+                snprintf(buffer, sizeof(buffer), "%d", arg_log_level);
+                setenv("CASYNC_LOG_LEVEL", buffer, 1);
```

Comment:
please either check for errors here, or eat up the errors explicitly by casting to (void). (in order to help static analyzers)

Reply:
Done for setenv, assuming it's not needed for snprintf after looking at other snprintf calls in the code.